src/redux.js

```diff
@@ -532,19 +544,27 @@ export function getAccuracy(state) {
       numCorrect++;
     }
   }
+  console.log("Number correct", numCorrect, "NumPredictedLabels", numPredictedLabels)
+  console.log('getAccuracy return value', ((numCorrect / numPredictedLabels) * 100).toFixed(2) )
   return ((numCorrect / numPredictedLabels) * 100).toFixed(2);
 }
 
+export function getAccuracyRegression(state) {
+  console.log('inside getAccuracyR Function')
+  let returnValue = getRange(state, state.labelColumn)
```

Comment:
I think we can call this variable `range` since `returnValue` is a little vague and might indicate it's the value `getAccuracyRegression` will return, which it's not.

Reply:
ok great! I'll  change the variable name to `range`